src/sentry/static/sentry/app/components/questionTooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {IconSize} from 'app/utils/theme';
+import Tooltip from 'app/components/tooltip';
+import {IconQuestion} from 'app/icons';
+
+type ContainerProps = {
+  className?: string;
+  size: IconSize | string;
+};
+
+const QuestionIconContainer = styled('span')<ContainerProps>`
+  display: inline-block;
+  height: ${p => p.theme.iconSizes[p.size] ?? p.size};
+  line-height: ${p => p.theme.iconSizes[p.size] ?? p.size};
+
+  & svg {
```

Comment:
Hmm I'm guessing `IconQuestion` isn't a styled component so we can't use it as a selector?

Tooltips get portaled, so any svgs in the tooltip wrapper shouldn't be affected, and there's no way to add svgs to the content, so this should be ok.

Reply:
Yeah tooltips will be portaled.